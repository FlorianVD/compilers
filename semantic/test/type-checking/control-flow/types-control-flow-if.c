int ifcond() {
    if (1) {
    } // this is OK
    if (1 == 1) {
    } // this is wrong
}
