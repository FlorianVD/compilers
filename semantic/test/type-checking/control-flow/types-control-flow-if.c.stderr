sema: error: Condition for an if statement must be an integer
